src/types/Chat.ts

```diff
@@ -0,0 +1,16 @@
+export enum CHAT_TYPES {
+  WORLD_CHAT = "WORLD_CHAT",
+  VENUE_CHAT = "VENUE_CHAT",
+  PRIVATE_CHAT = "PRIVATE_CHAT",
+}
+
+type SetPrivateChatTab = {
+  chatType: CHAT_TYPES.PRIVATE_CHAT;
+  recipientId: string | null;
```

Comment:
Why `null` here, and what does it give us over `undefined`? (see other comment in this review for my reasoning)